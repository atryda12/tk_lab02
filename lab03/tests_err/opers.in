
string s="", t="";
string u="";
int x="";

u = s-t;
u = s/t;



