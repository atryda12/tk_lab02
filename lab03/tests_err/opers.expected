
Error: Assignment of string to int: line 4
Error: Illegal operation, string - string: line 6
Error: Illegal operation, string / string: line 7
